apps/cookbook/src/app/showcase/segmented-control-showcase/segmented-control-showcase.component.html

```diff
@@ -1,8 +1,19 @@
 <div class="example">
-  <cookbook-example-viewer [html]="example.template">
-    <div>
-      <cookbook-segmented-control-example #example></cookbook-segmented-control-example>
-    </div>
+  <h3>Default</h3>
+  <cookbook-example-viewer [html]="example.template" [ts]="example.codeSnippet">
+    <cookbook-segmented-control-example-default
+      #example
+    ></cookbook-segmented-control-example-default>
+  </cookbook-example-viewer>
+  <h3>Grouped</h3>
```

Comment:
Consider adding an empty line between the two showcase types (improved readability)